custom-connectors/AzureADRiskyUsers/apiDefinition.swagger.json

```diff
@@ -0,0 +1,237 @@
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "userIds": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "description": "userIds",
+                    "title": "userIds"
+                  },
+                  "description": "userIds"
+                }
+              },
+              "default": {
+                "userIds": [
+                  "{{userID}}"
+                ]
+              }
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "204",
+            "schema": {}
+          }
+        }
+      }
+    },
+    "/v1.0/identityProtection/riskyUsers/{riskyUserId}": {
+      "get": {
+        "summary": "Get Risky user",
+        "description": "Get a single risky user",
+        "operationId": "GetRiskyUser",
+        "parameters": [
+          {
+            "name": "riskyUserId",
+            "in": "path",
+            "type": "string",
+            "required": true,
+            "description": "riskyUserId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "200",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "@odata.context": {
+                  "type": "string",
+                  "description": "@odata.context"
+                },
+                "id": {
+                  "type": "string",
+                  "description": "id"
+                },
+                "isDeleted": {
+                  "type": "boolean",
+                  "description": "isDeleted"
+                },
+                "isProcessing": {
+                  "type": "boolean",
+                  "description": "isProcessing"
+                },
+                "riskLevel": {
+                  "type": "string",
+                  "description": "riskLevel"
+                },
+                "riskState": {
+                  "type": "string",
+                  "description": "riskState"
+                },
+                "riskDetail": {
+                  "type": "string",
+                  "description": "riskDetail"
+                },
+                "riskLastUpdatedDateTime": {
+                  "type": "string",
+                  "description": "riskLastUpdatedDateTime"
+                },
+                "userDisplayName": {
+                  "type": "string",
+                  "description": "userDisplayName"
+                },
+                "userPrincipalName": {
+                  "type": "string",
+                  "description": "userPrincipalName"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/beta/identityProtection/riskyUsers/{filter}": {}
+  },
```

Comment:
@BakkerJan There is no metadata section in the swagger. Please add it, you can find instructions in the [documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#adding-metadata)

Reply:
Not sure if I did this correctly, but I added the metadata to the swagger file